### PR TITLE
Fixes issues with organization when saving the project form.

### PR DESCRIPTION
**Pull request by mabashian**
##### SUMMARY
Link #6086 

Screen Shot 2020-02-26 at 6 26 31 PM

The inventory form seemed to be functioning as expected after the `useField` updates so I changed the project form to match.

Also changes helperTextInvalid prop type to node which matches more closely with what the upstream PF component supports.

I noticed that while I was debugging #6086 an error was showing up in the console:

Screen Shot 2020-02-26 at 5 55 17 PM

In this case, the error was an array of strings.  I loosened the type checking on that field to (I think) match up with what the PF [component is expecting](https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-4/react-core/src/components/Form/FormGroup.tsx#L29).

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - UI

### CODE REVIEW

File: awx/ui_next/src/screens/Project/shared/ProjectForm.jsx [hunk 1]
```diff
@@ -164,10 +160,11 @@ function ProjectFormFields({
...
         onBlur={() => organizationHelpers.setTouched()}
         onChange={value => {
-          organizationHelpers.setValue(value.id);
-          setOrganization(value);
+          organizationHelpers.setValue(value);
         }}
-        value={organization}
+        value={organizationField.value}
+        touched={organizationMeta.touched}
+        error={organizationMeta.error}
```
Comment (marshmalien) [inline awx/ui_next/src/screens/Project/shared/ProjectForm.jsx]: Do we need to add these two props? It doesn't look like the OrganizationLookup uses touched or error, but I've seen this pattern in a few other places.

Comment (mabashian) [reply awx/ui_next/src/screens/Project/shared/ProjectForm.jsx]: @jlmitch5 maybe you can help shed some light on these props.  I followed patterns that I saw elsewhere with respect to configuring lookups but it doesn't look like `touched` or `error` is consumed by the lookup(s).  Should we remove these props or should we add logic to the lookups to consume them in some way?

Comment (jlmitch5) [reply awx/ui_next/src/screens/Project/shared/ProjectForm.jsx]: @marshmalien @mabashian when i was converting things from using Field -> useField, I was doing it based 100% on what was happening (i.e. being passed) before.  I didn't go in and check or change how they were be using.  It's probably that we got rid of the used of the touched/error props sometime, but didn't remove the props here.

Comment (marshmalien) [reply awx/ui_next/src/screens/Project/shared/ProjectForm.jsx]: >we got rid of the used of the touched/error props sometime, but didn't remove the props here

@mabashian @jlmitch5 Sounds like we can remove these props 👍

**Review by marshmalien — APPROVED** [review]
Tested this out locally. Everything worked as expected 👍

**Pull request merged**